stablehlo/reference/Ops.cpp

```diff
@@ -372,6 +403,56 @@ Tensor evalSliceOp(const Tensor &operand, Index startIndices, Sizes strides,
   return result;
 }
 
+SmallVector<Tensor> evalSortOp(ArrayRef<Tensor> inputs, Axis dimension,
+                               bool isStable, Region &comparator,
+                               Scope &scope) {
+  SmallVector<Tensor> runtimeResults;
+  for (auto input : inputs) runtimeResults.push_back(Tensor(input.getType()));
+  if (dimension < 0) dimension += inputs[0].getRank();
+  SmallVector<int64_t> indices(inputs.front().getShape()[dimension]);
+  auto inputShape = inputs.front().getShape();
+  inputShape[dimension] = 1;
+  auto inputSizes = Sizes(inputShape);
+  for (auto resultIt = IndexSpaceIterator(inputSizes, Index(inputShape.size()));
```

Comment:
Do no call the iterator as `resultIt` as this is not exactly result index at this point but a partial index with the dimension axis clipped to 0. We can use the `offsetIt` instead.